app/assets/javascripts/models/beta/iteration.js

```diff
@@ -0,0 +1,16 @@
+import moment from 'moment';
```

Comment:
I discussed that with @davidcostadev and maybe we could extract a function that simply calculates the sprint for a given date and project and then just reuse it for both `getSprint` and `getIterationForStory` (notice that we also standardized to always use the word `iteration` instead of `sprint` to avoid having two different terms for the same thing):

```js
const weeksBetween = (dateA, dateB) => moment(dateA).diff(moment(dateB), 'week');
   
const getIterationForDate = (date, project) => {
  const weeks = weeksBetween(date, project.startDate);
  const iterationQuantity = Math.ceil(weeks / project.iterationLength);

  return project.iterationLength > weeks ? iterationQuantity : iterationQuantity + 1;
};

export const getCurrentIteration = (project) => (
  getIterationForDate(new Date(), project)
);

export const getIterationForStory = (story, project) => {
  if (story.state === 'accepted') {
    return getIteractionQuantity(new Date(story.acceptedAt), project);
  }
}
```